packages/core/package-lock.json

```diff
@@ -1056,48 +1056,6 @@
         "rxjs": "^6.5.2"
       }
     },
-    "@deriv/p2p": {
```

Comment:
why is p2p removed from core package-lock?

Reply:
@kelly-binary since we import P2P into Core using a local link (`lerna link`), its existence in package-lock is unnecessary as it's not being pulled in from the npm registry. This removal is also automatic as this is an outdated entry, you can look at the `version` property at line 1060 and it shows that it's version `0.2.0` while the latest P2P in use is actually `0.5.1`. So this is just cleanup.

Reply:
ok, noted